package-lock.json

```diff
@@ -28,27 +28,28 @@
       }
```

Comment:
the changes in package-lock.json are because there were 2 critical vulnerabilities and `npm audit fix` got rid of them with the below changes